modules/nodes-cluster-resource-configure.adoc

```diff
@@ -11,6 +11,10 @@
 The Cluster Resource Override Operator requires a `ClusterResourceOverride` custom resource (CR)
 and a label for each project where you want the Operator to control overcommit.
 
+ifndef::openshift-rosa,openshift-dedicated[]
+By default, the installation process creates two Cluster Resource Override pods on the control plane nodes in the `clusterresourceoverride-operator` namespace. You can move these pods to other nodes, such as infrastructure nodes, as needed. Infrastructure nodes are not counted toward the total number of subscriptions that are required to run the environment. For more information, see the "Moving the Cluster Resource Override Operator pods" section that follows.
```

Comment:
In the event that further moving of modules occurs, consider dropping "section that follows."